source/Sashimi.Terraform.Tests/ActionHandlersFixture.cs

```diff
@@ -396,72 +396,52 @@ void PopulateVariables(TestActionHandlerContext<Program> _)
                 _.Variables.Add("bucket_name", bucketName);
                 _.Variables.Add(TerraformSpecialVariables.Action.Terraform.VarFiles, "example.tfvars");
                 _.Variables.Add(TerraformSpecialVariables.Action.Terraform.AWSManagedAccount, "AWS");
-                _.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, temporaryFolder);
+                _.Variables.Add(KnownVariables.OriginalPackageDirectoryPath, temporaryFolder.DirectoryPath);
             }
 
-            using (var outputs = ExecuteAndReturnLogOutput(PopulateVariables,
-                                                           "AWS",
-                                                           typeof(TerraformPlanActionHandler),
-                                                           typeof(TerraformApplyActionHandler),
-                                                           typeof(TerraformDestroyActionHandler))
-                .GetEnumerator())
-            {
-                outputs.MoveNext();
-                outputs.Current.OutputVariables.ContainsKey("TerraformPlanOutput").Should().BeTrue();
-
-                outputs.MoveNext();
-                outputs.Current.OutputVariables.ContainsKey("TerraformValueOutputs[url]").Should().BeTrue();
-                outputs.Current.OutputVariables["TerraformValueOutputs[url]"].Value.Should().Be($"https://{bucketName}.s3.amazonaws.com/test.txt");
+            var output = ExecuteAndReturnResult(typeof(TerraformPlanActionHandler), PopulateVariables, "AWS");
+            output.OutputVariables.ContainsKey("TerraformPlanOutput").Should().BeTrue();
 
-                string fileData;
-                using (var client = new HttpClient())
-                {
-                    fileData = await client.GetStringAsync($"https://{bucketName}.s3.amazonaws.com/test.txt").ConfigureAwait(false);
-                }
+            output = ExecuteAndReturnResult(typeof(TerraformApplyActionHandler), PopulateVariables, "AWS");
+            output.OutputVariables.ContainsKey("TerraformValueOutputs[url]").Should().BeTrue();
```

Comment:
These output variables disappear if you add `-state=xxx` as additional command line arguments like on [line 431](https://github.com/OctopusDeploy/Sashimi.Terraform/pull/2/files#diff-5173d5f99a2fe9258c85d0b0005adc90R431)